file_scanning/clamd/clamd.go

```diff
@@ -0,0 +1,229 @@
+package clamd
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"net/url"
+	"strings"
+)
+
+const (
+	RES_OK          = "OK"
+	RES_FOUND       = "FOUND"
+	RES_ERROR       = "ERROR"
+	RES_PARSE_ERROR = "PARSE ERROR"
+)
+
+type Clamd struct {
```

Comment:
Again should really be commented following the Go handbook
https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

`(can be ignored)`